data/scripts/libsoletta.sym

```diff
@@ -885,3 +885,12 @@ global:
 local:
         *;
 };
+
+LIBSOLETTA_2 {
+global:
+        sol_netctl_register_agent;
+        sol_netctl_unregister_agent;
+        sol_netctl_request_retry;
+        sol_netctl_request_input;
+        sol_netctl_scan;
+} LIBSOLETTA_1;
```

Comment:
these entries should be added with the commit that adds the function, otherwise we are broken between some commits.